Connectors/Revit/Speckle.Connectors.RevitShared/Operations/Receive/RevitHostObjectBuilder.cs

```diff
@@ -176,6 +194,12 @@ CancellationToken cancellationToken
     return new(bakedObjectIds, conversionResults);
   }
 
+  private void PreReceiveDeepClean(string baseGroupName)
+  {
+    _groupManager.PurgeGroups(baseGroupName);
```

Comment:
I would rename `_groupManager` to `_groupBaker` to make it aligned with class name